common/src/main/java/org/apache/comet/parquet/NativeBatchReader.java

```diff
@@ -214,7 +217,8 @@ private NativeBatchReader(AbstractColumnReader[] columnReaders) {
       boolean useLegacyDateTimestamp,
       StructType partitionSchema,
       InternalRow partitionValues,
-      Map<String, SQLMetric> metrics) {
+      Map<String, SQLMetric> metrics,
```

Comment:
Do we still need this parameter now? It was a placeholder for eventually implementing the metrics.